pkg/apis/node/v1alpha1/types.go

```diff
@@ -92,19 +92,35 @@ type Property struct {
 
 type CustomNodeResourceStatus struct {
 	// +optional
+	// +kubebuilder:deprecatedversion:warning="this field is deprecated and should be moved to Resources"
 	ResourceAllocatable *v1.ResourceList `json:"resourceAllocatable,omitempty"`
 
 	// +optional
+	// +kubebuilder:deprecatedversion:warning="this field is deprecated and should be moved to Resources"
 	ResourceCapacity *v1.ResourceList `json:"resourceCapacity,omitempty"`
 
+	Resources Resources `json:"resources"`
```

Comment:
Given that we already have `ResourceAllocatable` and `ResourceCapacity`, do we need the field `Resources`?

Reply:
Maybe we need to add an omitempty tag to this field?